### PR TITLE
add manual/ landing page

### PR DESCRIPTION
**Pull request by KaptenJansson**
* Added webrtc icon from webrtc/samples repo (for favicon)
* Added a separate css file for the landing page as I did not want to affect main.css.

Based off samples [landing page](https://webrtc.github.io/samples/)

### CODE REVIEW

File: src/manual/css/landing_page.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+a {
+color: #6fa8dc;
```
Comment (andresusanopinto) [inline src/manual/css/landing_page.css]: indent?

Comment (KaptenJansson) [inline src/manual/css/landing_page.css]: Ooops, I copied this from the samples repo and forgot to fix indendation (it's not indented at all there).

**Comment by KaptenJansson** [comment]
Also I left <h2> and <h3> styles for future use (currently not used).

**Comment by andresusanopinto** [comment]
Can you double check that /manual/ would show the index.html ? I think app.yaml might need some tweak for it to happen

**Comment by KaptenJansson** [comment]
Yeah indeed it does....Taking a look now.

**Comment by KaptenJansson** [comment]
PTAL

File: app.yaml [hunk 2]
```diff
@@ -12,6 +16,10 @@ handlers:
 - url: /manual
   static_dir: src/manual
 
+- url: /images/(.*\.(bmp|gif|ico|jpeg|jpg|png))
```
Comment (andresusanopinto) [inline app.yaml]: why use regex and not just upload the static files as is done already for manual ?

**Pull request merged**